bootstrap/kubeadm/.golangci.yml

```diff
@@ -1,21 +0,0 @@
-linters:
-  enable-all: true
-  disable:
-    - dupl
-    - funlen
-    - gochecknoglobals
-    - gochecknoinits
-    - gocyclo
-    - lll
-  fast: true
```

Comment:
How does this config align with the top level .golangci.yml file? iirc this one had more enabled linters, do we want to try to tighten up the top level one rather than relaxing the validation we are doing here?

Reply:
We can always align later?

Reply:
Can you create an issue to track that work?

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1552